src/test/java/org/mule/extension/sftp/SftpServer.java

```diff
@@ -35,7 +34,6 @@ public class SftpServer {
   public SftpServer(int port, Path path) {
     this.port = port;
     this.path = path;
-    configureSecurityProvider();
```

Comment:
Hi, @dipeshkumardutta-sfemu Just a question, did you analyse the impact in the non-fips environments? What could happen to those applications that were running in production without a FIPS-compliance Mule if we remove the functionality to register the security provider? Shouldn't it be conditional on whether or not the mule is in FIPS mode?

Reply:
This SftpServer class is inside test folder. This is a Sftp server used in junit testing. munit testcases are passing so I think the non-fips env testing is covered. Let me know if you think it has to be validated manually through a mule app.

Reply:
Sorry, forget it. I think I saw a security provider registration and all my alarms went off but I didn't realize where and what it was being used for.